optuna/integration/pytorch_distributed.py

```diff
@@ -93,9 +94,13 @@ class TorchDistributedTrial(optuna.trial.BaseTrial):
         trial:
             A :class:`~optuna.trial.Trial` object or :obj:`None`. Please set trial object in
             rank-0 node and set :obj:`None` in the other rank node.
-        device:
```

Comment:
The `device` argument is removed and this is distractive change. Do we need to consider the impact on users?

Reply:
Add an optional device argument. If device is not None, print warnings.